controllers/weather.controller.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+import { requestOptions } from "../utils/requestOptions.js";
+
+export class WeatherController {
+  static async getWeather(req, res) {
+    await axios
+      .request(requestOptions(req.query.cityName))
+      .then((response) => {
+        if (response.status === 200) {
+          res.status(200).json(response.data);
+        } else {
+          res.status(response.status).json(response.data);
+        }
+      })
+      .catch((error) => {
+        res.status(error.response.status).send(error.response.statusText);
```

Comment:
add log of error please

Reply:
and never return err to front-end - > return 500 error to client

Reply:
Responses fixed, as soon as I deal with `winston` I'll be able to implement good logs